src/ui/server/helpers/mappings/map-application-to-form-fields/index.test.ts

```diff
@@ -52,6 +54,52 @@ describe('server/helpers/mappings/map-application-to-form-fields', () => {
     });
   });
 
+  describe('when an application has exporterBusiness fields', () => {
+    it('should return the relevant exporterBusiness fields as timestamps', () => {
```

Comment:
There's no timestamps in `exporterBusiness`? Can just remove the "as timestamps" part